app/components/no_results_with_term/index.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {View} from 'react-native';
+
+import FormattedText from '@components/formatted_text';
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+import SearchIllustration from './search_illustration';
+
+type Props = {
+    term: string;
+};
+
+const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            flexGrow: 1,
+            alignItems: 'center' as const,
```

Comment:
why do we need the `as const` here?  I removed it and TS did not complained.

Reply:
Original code by @larkox I forgot why it was like that but is definitely TS related

Reply:
I don't remember how that ended up there, but I know the reason why I may have put it there at some point.

`alignItems` has a type like `'top' | 'center' | 'bottom' ...`. If you create a object `{alignItems: 'center'}` without specifying the type, that object cannot be assigned to a style, because `alignItems` is a string, and not `'top' | 'center' ...`. The way to tell javascript to type correctly alignItems is to say "here where I am using `'center'`, I'm not using a string, I am using a constant `center`.

Can be removed from here since getStyleFromTheme is poorly typed right now, but at some point we should look into making it so it returns proper types.